apps/bynder/src/index.js

```diff
@@ -237,6 +238,16 @@ setup({
       default: validAssetTypes.join(','),
       required: true,
     },
+    {
+      id: 'compactViewMode',
```

Comment:
Do we want this setting on global level? To me it feels like users want to configure this setting differently depending on the field.

Reply:
@andipaetzold are you thinking it would be set per field? Honestly, the reason I added the choice is if teams wanted to allow an editor to select a specific DAT from Bynder. The developer in me wouldn't want an editor to choose, I would want to make that choice for them.